dir.targets

```diff
@@ -154,5 +154,12 @@
 
   </Target>
 
+  <Target Name="SetPublicReleaseVersion" AfterTargets="GetBuildVersion">
+    <PropertyGroup Condition=" '$(PublicRelease)' == 'true' ">
+      <!-- Removes the prerelease tag from version when this is a public version but still has the git commit-->
+      <AssemblyInformationalVersion>$(BuildVersionSimple)$(SemVerBuildSuffix)</AssemblyInformationalVersion>
```

Comment:
Does NB.GV not do this automagically? That was my interpretation of the docs there . . .